tests/integration/cartography/intel/gitlab/test_members.py

```diff
@@ -0,0 +1,25 @@
+import cartography.intel.gitlab.members
+import tests.data.gitlab.group
+
+TEST_UPDATE_TAG = 123456789
+common_job_parameters={'UPDATE_TAG':'123456789',"WORKSPACE_ID":"123445","WORKSPACE_UUID":"234"}
+
+def test_load_member(neo4j_session):
+    cartography.intel.gitlab.members.load_members_data(
+        neo4j_session,
+        tests.data.gitlab.group.MEMBERS,
+        common_job_parameters,
+    )
+    nodes = neo4j_session.run(
+        """
+        MATCH (g:BitbucketMember) RETURN g.name;
```

Comment:
it says bitbucket member. can you please fix this?

Reply:
Done